python/ray/rllib/ppo/ppo.py

```diff
@@ -31,9 +34,11 @@
     # Initial coefficient for KL divergence
     "kl_coeff": 0.2,
     # Number of SGD iterations in each outer loop
-    "num_sgd_iter": 30,
+    "num_sgd_iter_policy": 30,
+    "num_sgd_iter_vf": 30,
```

Comment:
Number of SGD iterations for fitting the value function? (Or action-value function Q?)